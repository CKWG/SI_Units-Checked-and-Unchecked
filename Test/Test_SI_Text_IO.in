1.0 2.0*A 3.0*mA 4.0*mA**2 5.0*mA**(4/2) 6.0*A*kA**(-3)
7.0K
1.0*Bq 2.0*Ba no Ba
1.0E-3/mBqA no BqA
1.0*Gy 2.0*Gh no prefix on h
1.0*H  2.0*MH 3.0*Hz 4.0*MHz
1.0*lm 2.0*lx 3.0*ly no ly
1.0*m 2.0*dm 3.0*dam 4.0*cm 5.0*hm 6.0*mm 7.0*km 8.0*um 1.0*Mm
1.0*g 2.0*dag 3.0*kg
1.0*Pa 2.0*PPa
1.0*rad 2.0*sr
1.0*s
1.0*S 2.0*Sv
1.0*T 2.0*TT
1.0*W 2.0*Wb
1.0*Ohm
2.0*S*MOhm*Hz*s**2*uBq
1.0*ms**(-4/3)
1.0*km**(-1/2)
1.0*s**(-1)*m 2.0*s**-1*m  missing ()
1.0*mA**(3/2)*s
2.0*mA**3/2*s  missing ()
3.0*mA**3/s
1.0E-1/(dam**(-1)*s) 200.0*m/hs 3.0*km**2/s**2
1.0*m**/s  missing exp
5.0*kat 6.0*kkat 7.0*kd no kd
1.0*mmol 2.0*mol 3.0*mole too long
1.0/veryLongName_underline stops
1.0*l 2.0*hL 3.0/ml
-1.0*min 2.0*hmin no prefix allowed
1.0*h 2.0*mh no prefix allowed
1.0*W/(m*K)
6.674_2E-11*m**3/(kg*s**2)
9_ trailing underline
