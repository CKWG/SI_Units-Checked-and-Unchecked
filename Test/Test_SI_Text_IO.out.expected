 1.00000E+00
 2.00000E+00*A
 3.00000E-03*A
 4.00000E-06*A**2
 5.00000E-06*A**2
 6.00000E-09*A**(-2)
 7.00000E+00
ADA.IO_EXCEPTIONS.DATA_ERROR - rest of line: "K"
 1.00000E+00*s**(-1)
SI.ILLEGAL_UNIT - rest of line: " no Ba"
SI.ILLEGAL_UNIT - rest of line: " no BqA"
 1.00000E+00*m**2*s**(-2)
SI.ILLEGAL_UNIT - rest of line: " no prefix on h"
 1.00000E+00*m**2*kg*s**(-2)*A**(-2)
 2.00000E+06*m**2*kg*s**(-2)*A**(-2)
 3.00000E+00*s**(-1)
 4.00000E+06*s**(-1)
 1.00000E+00*cd
 2.00000E+00*m**(-2)*cd
SI.ILLEGAL_UNIT - rest of line: " no ly"
 1.00000E+00*m
 2.00000E-01*m
 3.00000E+01*m
 4.00000E-02*m
 5.00000E+02*m
 6.00000E-03*m
 7.00000E+03*m
 8.00000E-06*m
 1.00000E+06*m
 1.00000E-03*kg
 2.00000E-02*kg
 3.00000E+00*kg
 1.00000E+00*m**(-1)*kg*s**(-2)
 2.00000E+15*m**(-1)*kg*s**(-2)
 1.00000E+00
 2.00000E+00
 1.00000E+00*s
 1.00000E+00*m**(-2)*kg**(-1)*s**3*A**2
 2.00000E+00*m**2*s**(-2)
 1.00000E+00*kg*s**(-2)*A**(-1)
 2.00000E+12*kg*s**(-2)*A**(-1)
 1.00000E+00*m**2*kg*s**(-3)
 2.00000E+00*m**2*kg*s**(-2)*A**(-1)
 1.00000E+00*m**2*kg*s**(-3)*A**(-2)
 2.00000E+00
 1.00000E+04*s**(-4/3)
 3.16228E-02*m**(-1/2)
 1.00000E+00*m*s**(-1)
SI.ILLEGAL_UNIT - rest of line: "  missing ()"
 3.16228E-05*s*A**(3/2)
SI.ILLEGAL_UNIT - rest of line: "  missing ()"
 3.00000E-09*s**(-1)*A**3
 1.00000E+00*m*s**(-1)
 2.00000E+00*m*s**(-1)
 3.00000E+06*m**2*s**(-2)
SI.ILLEGAL_UNIT - rest of line: "  missing exp"
 5.00000E+00*s**(-1)*mol
 6.00000E+03*s**(-1)*mol
SI.ILLEGAL_UNIT - rest of line: " no kd"
 1.00000E-03*mol
 2.00000E+00*mol
SI.ILLEGAL_UNIT - rest of line: " too long"
SI.ILLEGAL_UNIT - rest of line: "_underline stops"
 1.00000E-03*m**3
 2.00000E-01*m**3
 3.00000E+06*m**(-3)
-6.00000E+01*s
SI.ILLEGAL_UNIT - rest of line: " no prefix allowed"
 3.60000E+03*s
SI.ILLEGAL_UNIT - rest of line: " no prefix allowed"
 1.00000E+00*m*kg*s**(-3)*K**(-1)
 6.67420E-11*m**3*kg**(-1)*s**(-2)
ADA.IO_EXCEPTIONS.DATA_ERROR - rest of line: " trailing underline"
